backend/apps/notifications/tasks.py

```
@@ -204,3 +204,158 @@ def send_notification_digests():
+
+    try:
+        success = channel_instance.deliver(delivery, delivery.recipient, payload)
+        if success:
+            delivery.status = "sent"
+            delivery.sent_at = timezone.now()
+            delivery.error_message = ""
+            delivery.save(update_fields=["status", "sent_at", "error_message", "updated_at"])
+            return True
+        else:
+            raise RuntimeError(f"Delivery returned False for channel '{channel_id}'")
+    except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)